api/build.gradle

```diff
@@ -508,7 +513,7 @@ dependencies {
     __tools__Implementation 'commons-cli:commons-cli:1.5.0'
     __tools__Implementation 'com.opencsv:opencsv:5.7.0'
 
-    testImplementation 'org.mockito:mockito-core:2.28.2'
+    testImplementation 'org.mockito:mockito-core:3.12.4'
```

Comment:
Required to bump Spring Boot.

Reply:
This ended up carrying a lot of baggage with it, so bumped in a [separate PR](https://github.com/all-of-us/workbench/pull/7623).